docs/deployment-environment-variables.md

```diff
@@ -90,37 +84,43 @@ SMTP_AUTHENTICATION="" # optional, "plain" by default
 SMTP_STARTTLS_AUTO=""  # optional, "true" by default
 ```
 
-### Rollbar
+### Time Zone
 
-Rollbar captures and notifies of errors in production, and requires a server-side access token.
+>HackathonManager will crash at startup if the time zone isn't valid.
+>
+>To find your appropriate time zone (e.g. `America/New_York`), see "TZ database name" on [Wikipedia](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
 
 ```bash
-ROLLBAR_ACCESS_TOKEN=""
+TIME_ZONE="America/Los_Angeles"
 ```
 
-### My MLH
+By default, charts & timestamps will be in UTC.
+
+## Optional
```

Comment:
Maybe add `The following environment variables are optional` just to be consistent w/ the description under `##Required`?